src/cli.cr

```diff
@@ -53,7 +55,7 @@ module Shards
       opts.on("--ignore-crystal-version", "Has no effect. Kept for compatibility, to be removed in the future.") { }
       opts.on("-v", "--verbose", "Increase the log verbosity, printing all debug statements.") { self.set_debug_log_level }
       opts.on("-q", "--quiet", "Decrease the log verbosity, printing only warnings and errors.") { self.set_warning_log_level }
-      opts.on("-h", "--help", "Print usage synopsis.") { self.display_help_and_exit(opts) }
+      opts.on("-h", "--help", "Print usage synopsis.") { display_help = true }
```

Comment:
Why's this change needed at all?

Reply:
This is explained in the commit itself: 5a6dc6cb28973f22758532a76e4585cb13969cfc

The block for `#on` will short circuit and the argument `--help` is never forwarded to the subcommand.

This is necessary for the subcommand to be able to have their own `--help` option.

Reply:
I see, thank you for the explanation.